content/get-involved.ts

```diff
@@ -27,37 +26,16 @@ const getInvolvedContent: GetInvolvedContent = {
     content: 'Join the Clearviction team and break down barriers for formerly incarcerated individuals by making it easier to vacate their criminal records in Washington State',
   },
   hero: {
-    header: 'Share your expertise with us',
-    subheading: 'There are many ways to participate with the Clearviction team, and we appreciate all of them!',
+    header: 'Get involved',
+    subheading: 'Conviction vacation makes it easier for formerly incarcerated individuals to access housing and employment, two things that are crucial for reducing the odds of re-incarceration.\n\n'
+    + 'By volunteering, you will help us reduce barriers and streamline the process of vacating eligible convictions in Washington state to make it easier for people to move forward.',
+    imgsrc: '../illustrations/team-building.svg',
+    ctaLink: '#open-roles',
```

Comment:
Will you please apply `useScroll` which can be found in the [CustomScroll.tsx](https://github.com/clearviction-devs/clearviction-wa/blob/main/components/functional/CustomScroll.tsx) file to this ctaLink. This might need to be done via the `HeroBanner` component. This scroll was created because the size of the navigation bar created a discrepancy in the x-scroll.